downstream/modules/platform/proc-creating-a-new-web-server-to-host-repositories.adoc

```diff
@@ -2,11 +2,11 @@
 
 = Creating a new web server to host repositories
 
-If you do not have an existing web server to host your repositories, create one with the synced repositories.
+If you do not have an existing web server to host your repositories, you can create one with your synced repositories.
 
 .Procedure
 
-Use the following steps if creating a new web server.
+Use the following steps to create a new web server.
 
 . Install prerequisites:
 +
```

Comment:
It might be worthwhile to check the other steps in this process: 

step 3. Is there an expected outcome after the user enters the command `sudo chown -R apache  /path/to/repos`? I see that the command for ensuring that the directory is readable is there, but what happens afterward? This can be ignored for now; just something to consider as we improve the docs.